PersonalBlogPlatform.Infrastructure/Repositories/PostsRepository.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.EntityFrameworkCore;
+using PersonalBlogPlatform.Core.Domain.Entities;
+using PersonalBlogPlatform.Core.Domain.RepositoryContracts;
+using PersonalBlogPlatform.Infrastructure.DbContext;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PersonalBlogPlatform.Infrastructure.Repositories
+{
+    public class PostsRepository : IPostsRepository
+    {
+
+        private readonly ApplicationDbContext _db; 
+        public PostsRepository ( ApplicationDbContext db)
+        {
+            _db = db;
+        }
+
+        public async Task<Post> AddPost(Post post)
+        {
+           _db.Posts.Add(post);
+           await _db.SaveChangesAsync();
+            return post;
+        }
+
+        public async Task<bool> DeletePostByPostId(Guid postId)
+        {
+            Post? post = await _db.Posts.FindAsync(postId);
+            if (post == null)
+                return false;
```

Comment:
Few changes should be done, repo should be granule ie one operation without much logic. Tomorrow will write detailed example. For now download dotnet clean architecture book and have a look

Reply:
hey, after few search, there is not much to be done, just continue